src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -117,6 +117,9 @@ struct sd_dhcp_client {
         sd_dhcp_lease *lease;
         usec_t start_delay;
         int ip_service_type;
+
+        /* ignore ifindex to generate iaid. See dhcp_identifier_set_iaid(). */
```

Comment:
ignore ifindex to generate → Ignore ifindex when generating

Reply:
Fixed. Thanks.